pixel-saver@deadalnix.me/decoration.js

```diff
@@ -122,6 +129,10 @@ function getOriginalState(win) {
 	if (win._pixelSaverOriginalState !== undefined) {
 		return win._pixelSaverOriginalState;
 	}
+
+	if (!win.decorated) {
```

Comment:
As this method gathers the initial state, the _GTK_HIDE_TITLEBAR_WHEN_MAXIMIZED flag should not have been set yet (at least not by PixelSaver). But either way, we want to record the original state. If a window is undecorated when PixelSaver is first invoked, we want to know.

This condition has been introduced to avoid that a window that has been initially undecorated become decorated when PixelSaver is disabled. `setHideTitlebar(win, false);` will only be invoked for the "DEFAULT" state (in function `disable()`).